frontend/src/app/task-area/task-area.component.html

```diff
@@ -0,0 +1,43 @@
+<div class="wrapper">
+  <div class="working-area">
+    <div class="side-bar">
+      <button class="btn active-tab">All</button>
+      <button class="btn">Groceries</button>
+      <button class="btn">College</button>
+      <button class="btn">Payments</button>
+    </div>
+    <div class="task-area">
+      <h1>All Tasks</h1>
+      <input
+        class="new-input"
+        type="text"
+        placeholder="Add a new task insdie ‘All’ category"
+        name=""
+        id=""
+      />
+      <div class="task-block">
+        <ul class="task-list">
+          <li class="task" *ngFor="let item of displayedTask">
+            <input
+              type="checkbox"
+              [checked]="item.status"
+              class="custom-checkbox"
+              id="taskDone"
+              name="happy"
+              value="yes"
+            />
+            <label for="taskDone"></label>
+            <span class="task-text">{{item.text}}</span>
+            <div class="task-tab">{{item.tabGroupe}}</div>
+            <div class="gaper"></div>
```

Comment:
not needed to past dummy element to align delete button to right. Use flex alignment